src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePumpOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using GuardNet;
+
+namespace Arcus.Messaging.Pumps.ServiceBus
+{
+    /// <summary>
+    ///     Options to configure how the Azure Service Bus message pump works
+    /// </summary>
+    public class AzureServiceBusMessagePumpOptions
+    {
+        private int? _maxConcurrentCalls;
+
+        /// <summary>
+        ///     Maximum concurrent calls to process messages
+        /// </summary>
+        public int? MaxConcurrentCalls
+        {
+            get => _maxConcurrentCalls;
+            set
+            {
+                Guard.For<ArgumentException>(() => value <= 0);
```

Comment:
May want to add an exception message here? Otheriwse we would get some global 'filter expression failure' exception message. Sorry for the lot of comments, btw.

Reply:
And you could use `Guard.NotLessThanOrEqualTo` for this. :)

Reply:
Added a message but NotLessThanOrEqualTo doesn't support nullables

Reply:
Ok